tensorflow/lite/tools/make/build_riscv_lib.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash 
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+set -x
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+TENSORFLOW_DIR="${SCRIPT_DIR}/../../../.."
+
+FREE_MEM="$(free -m | awk '/^Mem/ {print $2}')"
+# Use "-j 4" only memory is larger than 2GB
+if [[ "FREE_MEM" -gt "2000" ]]; then
+  NO_JOB=4
+else
+  NO_JOB=1
+fi
+
+make -j ${NO_JOB} TARGET=linux_riscv64 -C "${TENSORFLOW_DIR}" -f tensorflow/lite/tools/make/Makefile $@
```

Comment:
Don't you need to provide a way to build 32bit build?

Reply:
> Don't you need to provide a way to build 32bit build?

Actually, we tried on HiFive1 (32 bit MCU). I am getting issues with it.
Once it is fixed , will add support for RISC-V 32 bit MCU.

Reply:
I see. I think we might need to use an overridable variable to control this. You can do it later once it's ready.